sql/core/src/test/resources/sql-tests/inputs/nested-comments.sql

```diff
@@ -0,0 +1,9 @@
+-- This test case just used to test imported bracketed comments.
+
+-- the first case of bracketed comment
+--QUERY-DELIMITER-START
+/* This is the first example of bracketed comment.
+SELECT 'ommented out content' AS first;
```

Comment:
This is not a nested comment, right?

Reply:
This is a bracketed comments. 
Maybe I need change `nested-comments.sql` to `imported-comments.sql`?

Reply:
maybe updating `SQLQueryTestSuite` only is good enough. It's an obvious bug that we forget to handle `QUERY-DELIMITER` in imported code.

Reply:
You means I shouldn't change `comments.sql` and add `nested-comments.sql` ?

Reply:
yea, it's not necessary to separate this test file.

Reply:
OK. I will revert it.